integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java

```diff
@@ -23,6 +23,7 @@
 import oracle.weblogic.kubernetes.annotations.IntegrationTest;
 import oracle.weblogic.kubernetes.annotations.Namespaces;
 import oracle.weblogic.kubernetes.logging.LoggingFacade;
+import oracle.weblogic.kubernetes.utils.SessionMigrationUtil;
```

Comment:
We should create a new Utility Class MonitoringExporterUtils instead of using SessionMigrationUtil in MonitoringExporter usecase.  OR if this is a generic, we can add the method to CommonTestUtil class.

Reply:
I use this method from SessionMigrationUtil because it generates model file with unique domain name String modelFile = SessionMigrationUtil.generateSessionMigrYaml("ItMonitoringExporterSideCar", domain3Uid); Method is implemented to use specific file called model.sessmigr.yaml so I don't think it is generic, this is why I just reuse that method. Why I can't call utils I need in the test?

Reply:
Do not you think it can be made more generic, which can take any model (template) file not  only model.sessmigr.yaml ( hard-coded) to create a usable model file by replacing the domain id on the fly

Reply:
As I discussed with Marina, we can make the method generateSessionMigrYaml()  a generic utility  ( in a another tracking JIRA) which will take a template file and generate a usable model file based on the properties to be replaced